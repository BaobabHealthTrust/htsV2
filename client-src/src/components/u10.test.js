import React from 'react';
import ReactDOM from 'react-dom';
import U10 from './u10';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: U10', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <U10 />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})