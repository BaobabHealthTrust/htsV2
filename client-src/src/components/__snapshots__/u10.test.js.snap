// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: U10 should match its empty snapshot 1`] = `
<div
  className="u10"
>
  <table
    border="1"
    cellPadding="5"
    cellSpacing="0"
    style={
      Object {
        "border": "1px solid #345db5",
        "borderCollapse": "collapse",
        "width": "100%",
      }
    }
  >
    <tbody>
      <tr
        style={
          Object {
            "backgroundColor": "#345db5",
          }
        }
      >
        <td
          style={
            Object {
              "color": "#fff",
              "fontSize": "12px",
              "fontWeight": "bold",
              "paddingLeft": "10px",
            }
          }
        >
          Relationships
        </td>
      </tr>
      <tr>
        <td>
          <div
            className="u10TabContent"
          />
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
